### PR TITLE
Bump ubuntu snapshots

### PR DESCRIPTION
**Pull request by jovial**
Seems like we were missing the fixes for inception. Old snapshots had:

```
amd64-microcode (3.20191218.1ubuntu2.1) jammy-security; urgency=medium

  * SECURITY UPDATE: Zenbleed - information leak via speculative execution
    - microcode_amd_fam17h.bin{.asc}: update AMD fam17h cpu microcode and
      signature for Zenbleed vulnerability
    - New microcodes:
      + Family=0x17 Model=0x31 Stepping=0x00: Patch=0x0830107a Length=3200 bytes
      + Family=0x17 Model=0xa0 Stepping=0x00: Patch=0x08a00008 Length=3200 bytes
    - Updated microcodes:
      + Family=0x17 Model=0x08 Stepping=0x02: Patch=0x0800820d Length=3200 bytes
      + Family=0x17 Model=0x01 Stepping=0x02: Patch=0x0800126e Length=3200 bytes
    - CVE-2023-20593
```

but not:

```
amd64-microcode (3.20191218.1ubuntu2.2) jammy-security; urgency=medium

  * SECURITY UPDATE: INCEPTION - information leak via speculative execution
    - amd-ucode/microcode_amd_fam19h.bin{.asc}: add AMD fam19h cpu
      microcode and signature for Inception vulnerability
    - New microcodes:
      Family=0x19 Model=0x01 Stepping=0x00: Patch=0x0a001079 Length=5568 bytes
      Family=0x19 Model=0x01 Stepping=0x01: Patch=0x0a0011d1 Length=5568 bytes
      Family=0x19 Model=0x01 Stepping=0x02: Patch=0x0a001234 Length=5568 bytes
      Family=0x19 Model=0x11 Stepping=0x01: Patch=0x0a10113e Length=5568 bytes
      Family=0x19 Model=0x11 Stepping=0x02: Patch=0x0a10123e Length=5568 bytes
      Family=0x19 Model=0xa0 Stepping=0x01: Patch=0x0aa00116 Length=5568 bytes
      Family=0x19 Model=0xa0 Stepping=0x02: Patch=0x0aa00212 Length=5568 bytes
    - CVE-2023-20569

 -- Alex Murray <alex.murray@canonical.com>  Mon, 21 Aug 2023 10:25:31 +0930

```

### CODE REVIEW

**Comment by jovial** [comment]
Building new images here:

https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6121595206

**Comment by jovial** [comment]
I've merged in the centos versions bumps as there was a merge conflict

**Comment by jovial** [comment]
Promoting here: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6124518269

**Comment by jovial** [comment]
Hmm, missing containers in test pulp:

```
fatal: [controller0]: FAILED! => changed=true 
  msg: '''Traceback (most recent call last):\n  File "/opt/kayobe/venvs/kolla-ansible/lib/python3.8/site-packages/docker/api/client.py", line 268, in _raise_for_status\n    response.raise_for_status()\n  File "/opt/kayobe/venvs/kolla-ansible/lib/python3.8/site-packages/requests/models.py", line 960, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\nrequests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.43/images/create?tag=yoga-20230908T121709&fromImage=pulp-server.internal.sms-cloud%3A8080%2Fstackhpc-dev%2Fubuntu-source-fluentd\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/tmp/ansible_kolla_docker_payload_7tzuwse2/ansible_kolla_docker_payload.zip/ansible/modules/kolla_docker.py", line 406, in main\n  File "/tmp/ansible_kolla_docker_payload_7tzuwse2/ansible_kolla_docker_payload.zip/ansible/module_utils/kolla_docker_worker.py", line 656, in recreate_or_restart_container\n    self.start_container()\n  File
    "/tmp/ansible_kolla_docker_payload_7tzuwse2/ansible_kolla_docker_payload.zip/ansible/module_utils/kolla_docker_worker.py", line 674, in start_container\n    self.pull_image()\n  File "/tmp/ansible_kolla_docker_payload_7tzuwse2/ansible_kolla_docker_payload.zip/ansible/module_utils/kolla_docker_worker.py", line 455, in pull_image\n    json.loads(line.strip().decode(\''utf-8\'')) for line in self.dc.pull(\n  File "/opt/kayobe/venvs/kolla-ansible/lib/python3.8/site-packages/docker/api/image.py", line 430, in pull\n    self._raise_for_status(response)\n  File "/opt/kayobe/venvs/kolla-ansible/lib/python3.8/site-packages/docker/api/client.py", line 270, in _raise_for_status\n    raise create_api_error_from_http_exception(e)\n  File "/opt/kayobe/venvs/kolla-ansible/lib/python3.8/site-packages/docker/errors.py", line 31, in create_api_error_from_http_exception\n    raise cls(e, response=response, explanation=explanation)\ndocker.errors.NotFound: 404 Client Error for http+docker://localhost/v1.43/images/create?tag=yoga-20230908T121709&fromImage=pulp-server.internal.sms-cloud%3A8080%2Fstackhpc-dev%2Fubuntu-source-fluentd:
    Not Found ("manifest for pulp-server.internal.sms-cloud:8080/stackhpc-dev/ubuntu-source-fluentd:yoga-20230908T121709 not found: manifest unknown: Manifest not found.")\n'''
```

Trying manual sync here: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6170024506

Edit: I think this syncs for test pulp to ark, so probably no help ... I do have this container in my manifest:

```
REPOSITORY                                                                      TAG                    IMAGE ID       CREATED         SIZE
ark.stackhpc.com/stackhpc-dev/ubuntu-source-bifrost-deploy                      yoga-20230908T121709   524568a70d11   2 minutes ago   2.17GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-bifrost-base                        yoga-20230908T121709   7b6bde39eb97   2 minutes ago   2.16GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-compute                        yoga-20230908T121709   d1c2012f9077   2 hours ago     1.44GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-server                      yoga-20230908T121709   fbd22a6bae54   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cinder-volume                       yoga-20230908T121709   1b6df1e8a6a2   2 hours ago     1.33GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-horizon                             yoga-20230908T121709   39fa4f69e2d2   2 hours ago     1.1GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-novncproxy                     yoga-20230908T121709   c2dfe4c05426   2 hours ago     1.21GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cinder-backup                       yoga-20230908T121709   44200003d990   2 hours ago     1.32GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-l3-agent                    yoga-20230908T121709   852715dcee1f   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-dhcp-agent                  yoga-20230908T121709   5766e4c57265   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-sriov-agent                 yoga-20230908T121709   dfd4e6b94000   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-linuxbridge-agent           yoga-20230908T121709   34c564a2ef01   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-glance-api                          yoga-20230908T121709   3ae09656e0c9   2 hours ago     1.03GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-openvswitch-agent           yoga-20230908T121709   09059abb7625   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-metadata-agent              yoga-20230908T121709   0afb84eba3b1   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-neutron-agent                yoga-20230908T121709   149a2da281b2   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cinder-api                          yoga-20230908T121709   31688ee3873f   2 hours ago     1.29GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-ssh                            yoga-20230908T121709   6f180a3f16e9   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cinder-scheduler                    yoga-20230908T121709   d5901c7f4931   2 hours ago     1.29GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-blazar-api                          yoga-20230908T121709   678cfca1267c   2 hours ago     893MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-blazar-manager                      yoga-20230908T121709   d5cfa7b1a7c4   2 hours ago     893MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-neutron-base                        yoga-20230908T121709   9b710be50038   2 hours ago     1.23GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-spicehtml5proxy                yoga-20230908T121709   dd5c09c7c1c9   2 hours ago     1.15GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-compute-ironic                 yoga-20230908T121709   9ea6d7ad4ef7   2 hours ago     1.12GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-conductor                      yoga-20230908T121709   9d856ddd2dd9   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-api                         yoga-20230908T121709   e3f95927032f   2 hours ago     951MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-api                            yoga-20230908T121709   4a7bf7155248   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-scheduler                      yoga-20230908T121709   0b344cbcef86   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-driver-agent                yoga-20230908T121709   bfc4bc3b4d41   2 hours ago     951MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cinder-base                         yoga-20230908T121709   f87946b53db6   2 hours ago     1.29GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-conductor                    yoga-20230908T121709   75edf3ba95a8   2 hours ago     1.32GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-serialproxy                    yoga-20230908T121709   076ae388f70b   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-health-manager              yoga-20230908T121709   9186e4873aad   2 hours ago     924MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-backend-bind9             yoga-20230908T121709   44149e199aab   2 hours ago     907MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-housekeeping                yoga-20230908T121709   ad53cff3a53c   2 hours ago     924MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-worker                      yoga-20230908T121709   56e64cfbb8d4   2 hours ago     924MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-barbican-api                        yoga-20230908T121709   833b34de1ca1   2 hours ago     908MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-barbican-worker                     yoga-20230908T121709   8c614a213ecc   2 hours ago     908MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-manila-share                        yoga-20230908T121709   679d89d67503   2 hours ago     1.14GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-barbican-keystone-listener          yoga-20230908T121709   70867c3f4a42   2 hours ago     908MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-heat-engine                         yoga-20230908T121709   7b0eddb617e8   2 hours ago     959MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-heat-api                            yoga-20230908T121709   d89b75229b95   2 hours ago     959MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-heat-api-cfn                        yoga-20230908T121709   e19403e31894   2 hours ago     959MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cloudkitty-api                      yoga-20230908T121709   a3b44f2a1fbf   2 hours ago     899MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-worker                    yoga-20230908T121709   aea4ac53a921   2 hours ago     907MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cloudkitty-processor                yoga-20230908T121709   46ef6595ab18   2 hours ago     899MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-placement-api                       yoga-20230908T121709   a4fa7949dbaa   2 hours ago     892MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-keystone-ssh                        yoga-20230908T121709   09bd8c61283a   2 hours ago     950MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-central                   yoga-20230908T121709   1af42c95caa0   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-producer                  yoga-20230908T121709   c0b0614a3697   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-keystone-fernet                     yoga-20230908T121709   7cbbe57b26ea   2 hours ago     948MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-api                       yoga-20230908T121709   67f73627c5e2   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-sink                      yoga-20230908T121709   bc6998ab563d   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-mdns                      yoga-20230908T121709   a937e99ae275   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-pxe                          yoga-20230908T121709   25d72c7ccaad   2 hours ago     985MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-keystone                            yoga-20230908T121709   7e79750d3e06   2 hours ago     943MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-manila-data                         yoga-20230908T121709   43f24b5be7ce   2 hours ago     957MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-manila-api                          yoga-20230908T121709   f0c3ec71526c   2 hours ago     953MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-manila-scheduler                    yoga-20230908T121709   8eddce910b68   2 hours ago     953MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-blazar-base                         yoga-20230908T121709   3503bee1e27b   2 hours ago     893MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-magnum-conductor                    yoga-20230908T121709   66c33ffd8ac4   2 hours ago     974MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-magnum-api                          yoga-20230908T121709   29314daf6aa1   2 hours ago     974MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-glance-base                         yoga-20230908T121709   b828c9a14d3e   2 hours ago     1.02GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-api                          yoga-20230908T121709   29a95c25ef82   2 hours ago     929MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-inspector                    yoga-20230908T121709   105d94445a21   2 hours ago     899MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-base                           yoga-20230908T121709   d2ded54ce5fd   2 hours ago     1.11GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-octavia-base                        yoga-20230908T121709   c9c6d48c240a   2 hours ago     924MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-barbican-base                       yoga-20230908T121709   a3904b0f68d6   2 hours ago     908MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-heat-base                           yoga-20230908T121709   a785111e660e   2 hours ago     959MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cloudkitty-base                     yoga-20230908T121709   d5122d661aa8   2 hours ago     899MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-placement-base                      yoga-20230908T121709   9adec706a685   2 hours ago     892MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-keystone-base                       yoga-20230908T121709   64fb5116dc7b   2 hours ago     941MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-designate-base                      yoga-20230908T121709   0be215b6d22b   2 hours ago     900MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-caso                                yoga-20230908T121709   c834745f1bd0   2 hours ago     896MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-kolla-toolbox                       yoga-20230908T121709   7aed78863f14   2 hours ago     804MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-manila-base                         yoga-20230908T121709   eb3603c7d14f   2 hours ago     953MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-magnum-base                         yoga-20230908T121709   e2f9bc81bc8e   2 hours ago     974MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ironic-base                         yoga-20230908T121709   41aba4da96a6   2 hours ago     929MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-libvirt-exporter         yoga-20230908T121709   fe4aa92bc3b7   2 hours ago     863MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-openstack-base                      yoga-20230908T121709   bade064a4092   2 hours ago     885MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-fluentd                             yoga-20230908T121709   fb587cdb9637   2 hours ago     553MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-mariadb-server                      yoga-20230908T121709   9a7348214ec9   2 hours ago     617MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ovn-sb-db-server                    yoga-20230908T121709   0ce5b52807f7   2 hours ago     272MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ovn-controller                      yoga-20230908T121709   a755ea68d6cc   2 hours ago     271MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ovn-nb-db-server                    yoga-20230908T121709   d64b3b17f2d6   2 hours ago     272MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-mariadb-clustercheck                yoga-20230908T121709   2df3bf1a8b7b   2 hours ago     304MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ovn-northd                          yoga-20230908T121709   3e1777c6abe0   2 hours ago     272MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-skydive-agent                       yoga-20230908T121709   004d8b062056   2 hours ago     408MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-redis-sentinel                      yoga-20230908T121709   57dd2a6f0b1b   2 hours ago     242MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-redis                               yoga-20230908T121709   bc30e1cfc303   2 hours ago     242MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-cadvisor                 yoga-20230908T121709   2ebc081afb87   2 hours ago     276MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-skydive-analyzer                    yoga-20230908T121709   ea164589a4c2   2 hours ago     400MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-nova-libvirt                        yoga-20230908T121709   96535ee97d30   2 hours ago     1GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-memcached-exporter       yoga-20230908T121709   e59093b2a6e4   2 hours ago     250MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-msteams                  yoga-20230908T121709   ca6a709bb8c2   2 hours ago     260MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-haproxy-exporter         yoga-20230908T121709   5c272ee751d8   2 hours ago     251MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-jiralert                 yoga-20230908T121709   e1e62675617d   2 hours ago     706MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-v2-server                yoga-20230908T121709   ef5a6ec15dc3   2 hours ago     440MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-elasticsearch-exporter   yoga-20230908T121709   e4dc346e8de8   3 hours ago     247MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-openstack-exporter       yoga-20230908T121709   1b5b24d2dc78   3 hours ago     247MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-mtail                    yoga-20230908T121709   b04f8fe569c5   3 hours ago     252MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-mysqld-exporter          yoga-20230908T121709   272019d593b5   3 hours ago     252MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-grafana                             yoga-20230908T121709   9e02bb669af2   3 hours ago     620MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-node-exporter            yoga-20230908T121709   76d509abf0bd   3 hours ago     256MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-blackbox-exporter        yoga-20230908T121709   8b21385cf98f   3 hours ago     273MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-alertmanager             yoga-20230908T121709   0362b7390b81   3 hours ago     294MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-elasticsearch                       yoga-20230908T121709   b71c3f92d1a3   3 hours ago     866MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-ovn-base                            yoga-20230908T121709   149b5c9cb475   3 hours ago     267MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-openvswitch-netcontrold             yoga-20230908T121709   b5e19f847183   3 hours ago     273MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-openvswitch-db-server               yoga-20230908T121709   96ea62c08d0b   3 hours ago     256MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-rabbitmq                            yoga-20230908T121709   be4af1aaf9dd   3 hours ago     292MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-openvswitch-vswitchd                yoga-20230908T121709   46bb19f72835   3 hours ago     256MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-etcd                                yoga-20230908T121709   ba5c7fd5c756   3 hours ago     276MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-keepalived                          yoga-20230908T121709   32a5e65c91af   3 hours ago     256MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-mariadb-base                        yoga-20230908T121709   c70940f555c3   3 hours ago     302MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-cron                                yoga-20230908T121709   4d45daf6b95d   3 hours ago     238MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-kibana                              yoga-20230908T121709   58215412e390   3 hours ago     567MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-dnsmasq                             yoga-20230908T121709   90786675a160   3 hours ago     239MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-elasticsearch-curator               yoga-20230908T121709   850b181f02ce   3 hours ago     347MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-skydive-base                        yoga-20230908T121709   af4e25011100   3 hours ago     400MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-redis-base                          yoga-20230908T121709   679b94000c2f   3 hours ago     237MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-haproxy                             yoga-20230908T121709   1e0d142476e0   3 hours ago     246MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-opensearch                          yoga-20230908T121709   67ba77e11aa3   3 hours ago     1.27GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-opensearch-dashboards               yoga-20230908T121709   b92faf510453   3 hours ago     1.35GB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-prometheus-base                     yoga-20230908T121709   da638afc4aa2   3 hours ago     237MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-openvswitch-base                    yoga-20230908T121709   a40d430e1321   3 hours ago     256MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-influxdb                            yoga-20230908T121709   8b0f47bbf49f   3 hours ago     391MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-memcached                           yoga-20230908T121709   d61ff96bad57   3 hours ago     239MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-tgtd                                yoga-20230908T121709   46f27d5cfe06   3 hours ago     237MB
ark.stackhpc.com/stackhpc-dev/ubuntu-source-base                                yoga-20230908T121709   fd1a66ef3b22   3 hours ago     237MB
```

**Comment by jovial** [comment]
I can only think that I must not have selected the "push images " option. Building again:

https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6170125945

**Comment by jovial** [comment]
Rebuilding after test-pulp was restored: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6174932225

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -6,58 +6,64 @@
...
 
+opensearch_tag: yoga-20230904T121340
+
 kayobe_image_tags:
   openstack:
-    centos: yoga-20230217T135826
+    centos: yoga-20230905T130221
     rocky: yoga-20230310T170929
-    ubuntu: yoga-20230220T181235
+    ubuntu: yoga-20230908T121709
```
Comment (jovial) [inline etc/kayobe/kolla/globals.yml]:
```suggestion
    ubuntu: yoga-20230913T160015
```

**Comment by jovial** [comment]
Promoting images here: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6199616595

File: etc/kayobe/environments/ci-builder/stackhpc-ci.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ kolla_enable_etcd: true
 kolla_enable_grafana: true
 kolla_enable_influxdb: true
 kolla_enable_ironic: true
+kolla_enable_kibana: "{{ kolla_base_distro not in ['rocky'] }}"
```
Comment (jovial) [inline etc/kayobe/environments/ci-builder/stackhpc-ci.yml:21]: Found kibana images was missing ...

**Comment by jovial** [comment]
- intel-microcode (3.20230214.0ubuntu0.22.04.1) -> intel-microcode (3.20230808.0ubuntu0.16.04.1+esm1)
- amd64-microcode (3.20191218.1ubuntu2.1)  -> amd64-microcode (3.20191218.1ubuntu2.2)

File: releasenotes/notes/bump-ubuntu-snapshots-2023-09-15-22ca5250d40bd5b6.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+security:
+  - |
+    Bumps Ubuntu repository snapshots and container images to bring in latest
+    security patches. This includes the microcode to patch Inception
+    (CVE-2023-20569) and Downfall (CVE-2022-40982). Zenbleed (CVE-2023-20593)
+    was patched in the previous snapshot bump.
```
Comment (markgoddard) [inline releasenotes/notes/bump-ubuntu-snapshots-2023-09-15-22ca5250d40bd5b6.yaml]: Are these resolved via package update and reboot?

Comment (jovial) [reply releasenotes/notes/bump-ubuntu-snapshots-2023-09-15-22ca5250d40bd5b6.yaml]: That is the recommended approach. There is a feature called late loading, but this is not recommended:

https://docs.kernel.org/arch/x86/microcode.html#late-loading

Comment (jovial) [reply releasenotes/notes/bump-ubuntu-snapshots-2023-09-15-22ca5250d40bd5b6.yaml]: I've added a statement to say that the host needs rebooting.

**Comment by jovial** [comment]
Hit issues with the last run as ubuntu images still missing in test pulp. Will try a manual sync again.

Edit: possibly it was this sync that hit the 6h timeout: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6199502513

new sync: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6233132623

**Review by markgoddard — APPROVED** [review]
Nice perseverance!

**Comment by bbezak** [comment]
how about opensearch image? 
https://github.com/stackhpc/stackhpc-kayobe-config/pull/630/files#diff-0171daf65a8de17528b885bebd2348f90778db14dcc5e27354254a537d2b2d12R13
@markgoddard @jovial

**Comment by jovial** [comment]
> how about opensearch image? https://github.com/stackhpc/stackhpc-kayobe-config/pull/630/files#diff-0171daf65a8de17528b885bebd2348f90778db14dcc5e27354254a537d2b2d12R13 @markgoddard @jovial

Didn't bump those because I rebased on: https://github.com/stackhpc/stackhpc-kayobe-config/pull/618. Do we need to bump them?

**Comment by bbezak** [comment]
well, opensearch image that I built didn't use the new ubuntu snapshot?

**Comment by bbezak** [comment]
> > how about opensearch image? https://github.com/stackhpc/stackhpc-kayobe-config/pull/630/files#diff-0171daf65a8de17528b885bebd2348f90778db14dcc5e27354254a537d2b2d12R13 @markgoddard @jovial
> 
> Didn't bump those because I rebased on: #618. Do we need to bump them?

well, opensearch image that I built didn't use the new ubuntu snapshot?

**Comment by markgoddard** [comment]
> > > how about opensearch image? https://github.com/stackhpc/stackhpc-kayobe-config/pull/630/files#diff-0171daf65a8de17528b885bebd2348f90778db14dcc5e27354254a537d2b2d12R13 @markgoddard @jovial
> > 
> > 
> > Didn't bump those because I rebased on: #618. Do we need to bump them?
> 
> well, opensearch image that I built didn't use the new ubuntu snapshot?

I think we still need this fix?

**Pull request merged**